kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -39,7 +39,7 @@ def _get_synapse_by_name(self, synapse_name):
         """
         all_synapse = self.brain.brain_yaml
         for el in all_synapse:
-            print el
+            print(el)
```

Comment:
The python3 interpreter does not allow this old syntax, this one is compatible with both python2 and python3.